torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -158,7 +158,7 @@ class OpConfig {
         inputs.begin(), inputs.end(), [](const at::Tensor& tensor) {
           return bridge::TryGetXlaTensor(tensor);
         });
-    XLA_CHECK(it != inputs_.end());
+    XLA_CHECK(it != inputs.end());
```

Comment:
Previously, incorrectly checking against `inputs_` instead of `inputs`.

Reply:
man! I should catch this during reviewing haha

Reply:
Must admit that it was the best naming choice, though.

Reply:
nice catch 🤣 , I will check the TPU CI after this pr is merged.

Reply:
Again, sorry for the typo there. That was really bad.